### ISSUE TITLE
[Question] Preventing code blocks by tabs

### ISSUE THREAD

**Issue by HarunD** [russross/blackfriday#537]
I am working on a Hugo website and I am wondering whether there is a way to disable the default behavior that converts tabbed content to code blocks.

I have a case where I would like to use only code fences and therefore disable/prevent the behavior that converts tabbed content into code blocks.

I am aware that this is a default behvaiour for markdown and see that there is no option currently to configure Blackfriday to disable tabs-to-code block conversion - I would just like to see whether anyone had a similar requirement and whether there is a straightforward way to do it.

Thanks!

**Comment by rtfb** [comment]
Yeah, a cursory look at the flags confirms there's no way to do that. And I'm not aware of any good workaround.

In v2 you could possibly post-process non-fenced code block elements and custom-render them as if they're not code, but that won't help you with Hugo.

**Comment by HarunD** [comment]
@rtfb thanks for your answer
